src/components/operations/Composition.js

```diff
@@ -52,9 +53,8 @@ class Composition extends PureComponent {
           </FormGroup>
         </div>
         <div className="col-md-6">
-          <FormGroup label="Secondary color" required={this.isRequired('secondary_color')} error={errors.secondary_color}>
-            <Input name="secondary_color" type="text" className="rka-input-txt" placeholder="secondary color"
-              value={values.secondary_color} defaultValue={defaults.secondary_color.default} onChange={this.props.onChange} />
+          <FormGroup label="Secondary Color" required={this.isRequired('secondary_color')} error={errors.color}>
+            <InputColor name="secondary_color" value={values.secondary_color || '000000'} onChange={this.props.onChange} />
```

Comment:
the default should be taken from the API. We have it in props somewhere, let's look at it together.